### PR TITLE
AQ-28999 Updates for AQTS 22.3 service models changes

### PR DESCRIPTION
**Pull request by josef-mixan-ai**

### CODE REVIEW

File: src/Aquarius.Client/TimeSeries/Client/ServiceModels/Acquisition.cs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /* Options:
-Date: 2022-08-18 21:02:14
+Date: 2022-10-25 21:44:15
 Version: 5.104
 Tip: To override a DTO option, remove "//" prefix before updating
-BaseUrl: https://aqts-rel-pg.aquariusdev.net/AQUARIUS/Acquisition/v2
+BaseUrl: http://aqts-pg.aquariusdev.net/AQUARIUS/Acquisition/v2
```
Comment (AlvinLee-AI) [inline src/Aquarius.Client/TimeSeries/Client/ServiceModels/Acquisition.cs:5]:
```suggestion
BaseUrl: https://aqts-pg.aquariusdev.net/AQUARIUS/Acquisition/v2
```

File: src/Aquarius.Client/TimeSeries/Client/ServiceModels/Provisioning.cs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /* Options:
-Date: 2022-08-18 21:02:04
+Date: 2022-10-25 21:44:14
 Version: 5.104
 Tip: To override a DTO option, remove "//" prefix before updating
-BaseUrl: https://aqts-rel-pg.aquariusdev.net/AQUARIUS/Provisioning/v1
+BaseUrl: http://aqts-pg.aquariusdev.net/AQUARIUS/Provisioning/v1
```
Comment (AlvinLee-AI) [inline src/Aquarius.Client/TimeSeries/Client/ServiceModels/Provisioning.cs:5]:
```suggestion
BaseUrl: https://aqts-pg.aquariusdev.net/AQUARIUS/Provisioning/v1
```

File: src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs [hunk 3]
```diff
@@ -1,8 +1,8 @@
 /* Options:
-Date: 2022-08-18 21:01:54
+Date: 2022-10-25 21:44:12
 Version: 5.104
 Tip: To override a DTO option, remove "//" prefix before updating
-BaseUrl: https://aqts-rel-pg.aquariusdev.net/AQUARIUS/Publish/v2
+BaseUrl: http://aqts-pg.aquariusdev.net/AQUARIUS/Publish/v2
```
Comment (AlvinLee-AI) [inline src/Aquarius.Client/TimeSeries/Client/ServiceModels/Publish.cs:5]:
```suggestion
BaseUrl: https://aqts-pg.aquariusdev.net/AQUARIUS/Publish/v2
```

**Review by AlvinLee-AI — APPROVED** [review]
Looks fine to me. Not sure if the `baseUrl`s were manually changed, or if that's what was generated from running the create service models script.

**Comment by josef-mixan-ai** [comment]
@AlvinLee-AI thank you for reviewing.

It doesn't matter which server is used neither over which protocol, as long as the server has the desired AQTS release candidate version deployed. In this case, the rel-pg server was turned off, probably to save some budget. I used a mixture of my local VM and aqts-pg.

**Pull request merged**